vaccine_feed_ingest/runners/in/arcgis/normalize.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+# import schema
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
+
+from schema import schema  # noqa: E402
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("ak/arcgis/normalize.py")
+
+
+def _get_id(site: dict) -> str:
+    data_id = site["attributes"]["GlobalId"]
+
+    # Could parse these from directory traversal, but do not for now to avoid
+    # accidental mutation.
+    site = "arcgis"
+    runner = "in"
+
+    # Could parse these from the input file name, but do not for now to avoid
+    # accidental mutation.
+    arcgis = "97135bbb1bec488e9717aca061c03e41"
+    layer = 0
+
+    return f"{runner}:{site}:{arcgis}_{layer}:{data_id}"
+
+
+def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
+    vaccines_field = site["attributes"]["Vaccine_Type"].lower().split(",")
+
+    potentials = {
+        "Pfizer": schema.Vaccine(vaccine="pfizer"),
+        "Moderna": schema.Vaccine(vaccine="moderna"),
+        "Johnson & Johnson": schema.Vaccine(vaccine="janssen"),
+        "FRPP": schema.Vaccine(vaccine="frpp"),
+    }
+
+    inventory = []
+
+    for vf in vaccines_field:
+        try:
+            inventory.append(potentials[vf])
+        except KeyError as e:
+            logger.error("Unexpected vaccine type: %s", e)
+
+    if len(inventory) > 0:
+        return inventory
+
+    return None
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    contacts = []
+    if site["attributes"]["Site_Phone"]:
+        sourcePhone = re.sub("[^0-9]", "", site["attributes"]["Site_Phone"])
+        if len(sourcePhone) == 11:
+            sourcePhone = sourcePhone[1:]
+        phone = f"({sourcePhone[0:3]}) {sourcePhone[3:6]}-{sourcePhone[6:]}"
+        contacts.append(schema.Contact(phone=phone))
+
+    if site["attributes"]["Site_Zotec_Link"]:
+        contacts.append(schema.Contact(website=site["attributes"]["Site_Zotec_Link"]))
+
+    if site["attributes"]["Site_Location_Info"]:
+        contacts.append(schema.Contact(other=site["attributes"]["Site_Location_Info"]))
+
+    if len(contacts) > 0:
+        return contacts
+
+    return None
+
+
+def _get_notes(site: dict) -> Optional[List[str]]:
+    if site["attributes"]["Site_Special_Inst"]:
```

Comment:
It looks like there may also be notes in `Site_Location_Info`. Could we add those?